src/requirements/Allowlist/AllowlistRequirement.tsx

```diff
@@ -48,7 +48,18 @@ const AllowlistRequirement = ({ ...rest }: RequirementProps): JSX.Element => {
       {hideAllowlist ? (
         `${isEmail ? "email " : ""}allowlist`
       ) : (
-        <Button variant="link" rightIcon={<ArrowSquareIn />} onClick={onOpen}>
+        <Button
+          variant="link"
+          rightIcon={<ArrowSquareIn />}
+          {...("ipfsHash" in requirement.data
```

Comment:
I wonder if we should dynamically load the `SearchableVirtualListModal` too only in case when we don't use this new type of allowlist? Just to optimize this requirement even more 🔥

Reply:
Great idea! Changed in https://github.com/guildxyz/guild.xyz/pull/1149/commits/2a2c753cdecda617b6211b0d3ad0f44487a42ad0, seems to work well 👍